tools/update_test_data.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+# Import utility functions
+# shellcheck source=./tools/_functions.sh
+source "$(dirname "${BASH_SOURCE[0]}")/_functions.sh"
+
+require_installed
+require_database
+
+deescalate_privileges integreat-cms-cli dumpdata --natural-primary --natural-foreign --exclude auth --exclude contenttypes --o test_data.json
```

Comment:
Did not test this yet, but I think the output should replace the existing test data file at `integreat_cms/cms/fixture/test_data.json`. Right now it only creates a new file in the root directory.

Reply:
Thanks!